src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url("../images/star.svg") no-repeat;
+    align-items: center;
+    background-position: center;
+
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(1),
+  &__star:nth-child(-n+2),
+  &__star:nth-child(-n+3),
+  &__star:nth-child(-n+4) {
+    background: url("../images/star-active.svg") center no-repeat;
+  }
```

Comment:
Not done yet: `all `stars--N` modifiers work as expected (Highlight first `N` stars)`. E.g `stars--1` and `stars--3` should work like in previous `stars` task